### DISCUSSION TITLE
"10.0.6" and the latest version "^10.0.7-canary.2" not working with ie11 version 18363.1316

### DISCUSSION THREAD

**Discussion by lanyusan** [vercel/next.js#21797] [Help]
**What version of Next.js are you using?**

"^10.0.7-canary.2", and also 10.0.6

**What version of Node.js are you using?**

v14.15.0

**What browser are you using?**

ie 11  version 18363.1316

**What operating system are you using?**

windows

**How are you deploying your application?**

next dev

**Describe the Bug**

Created a naked nextjs app with `yarn create next-app` and used `npm install next@canary` to upgrade to latest. 

Run server with `yarn dev`
Open `http://localhost:3000` with ie 11

It shows a blanket page and dev tool shows errors:

`SCRIPT1014: 无效字符 (invalid character)`, which points to` _app.js (501,487)`

this line:

`count=idCountMap[id]||0;const identifier=`${id} ${count}`;idCountMap[id]=count+1;const index=getIndexByIdentifier(identifier);const obj={css:item[1],media:item[2],sourceMap:item[3]};if(index!==-1){stylesInDom[index].references+ ...`

**Expected Behavior**

Nextjs standard app page should display correctly at ie11 as claimed by nextjs that it works out of box with ie11

**To Reproduce**

`yarn create next-app`

`npm install next@canary`

`yarn dev`

open `http://localhost:3000` with ie11

**Comment by uasi** [comment]
Related issue https://github.com/vercel/next.js/issues/21130 (closed, but I believe it is not fixed yet)
